hyperSpec/vignettes/hyperSpec.Rmd

```diff
@@ -373,19 +373,19 @@ The second argument of the extraction functions `[]` and `[[]]` specifies the ex
 They can be given like any column specification for a `data.frame`{.r}, i.e., numeric, logical, or by a vector of the column names.  However, since there is intrinsic order the column names of a `hyperSpec` object, using the column names is safest:
 
 ```{r data}
-colnames(chondro)
+colnames(faux_cell)
 ```
-```{r data-chondro-b}
-chondro[[1:3, 1]]
+```{r data-faux_cell-b}
```

Comment:
Code chunk names should not contain any underscores. As far as I know, only (non-capital??) letters, digits and minus signs (`-`) are allowed.